File: level1_ecitizen.cpp
// Level 1 - Login Validation (C++)
#include <iostream>
using namespace std;

int main() {
    string correctUsername = "adminKE";
    string correctPassword = "254Secure";

    string username, password;

    cout << "Enter username: ";
    cin >> username;

    cout << "Enter password: ";
    cin >> password;

    if (username == correctUsername && password == correctPassword) {
        cout << "Access Granted";
    } else {
        cout << "Invalid Credentials";
    }

    return 0;
}

File: level1_ecitizen.py
# Level 1 - Login Validation (Python)

# Correct credentials
correct_username = "adminKE"
correct_password = "254Secure"

# Ask user for input
username = input("Enter username: ")
password = input("Enter password: ")

# Check credentials
if username == correct_username and password == correct_password:
    print("Access Granted")
else:
    print("Invalid Credentials")